client/src/components/Requests/RequestMenu/RequestMenu.tsx

```diff
@@ -6,35 +6,40 @@ import SettingsIcon from '@material-ui/icons/Settings';
 import React from 'react';
 import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
+import { useRequest } from '../../../context/useRequestContext';
 
 interface Props {
   request: Request | null | undefined;
 }
 
 const RequestMenu = ({ request }: Props): JSX.Element => {
+  const { sendResponse } = useRequest();
   const classes = useStyles();
-  const accepted = request?.accepted == true;
   const [menuToggle, setmenuToggle] = React.useState<null | HTMLElement>(null);
+  if (request) {
+    const accepted = request.accepted == true;
```

Comment:
Prefer to use strict equality operator `===`. Besides, since `request.accepted` is already a boolean, I don't see any need to reassign it to a new variable.

Reply:
Change made. No need for it.

Reply:
I had to go back here...type error:

request.accepted is currently: `true | false | null` to identify status of booking (true,false,undecided)
disabled prop only takes `true | false | undefined`

This is why I defined the boolean, in case 'request.accepted' is null.
 
I tried to change to model to make the default value 'undefined' but  MongoDB will simply not assign accepted if the default is 'undefined'. Then will then be no 'request.accepted' key for the component to read and it will need a different check for that being missing rather than 'null'.